showcase/shell-docs/src/app/globals.css

```
@@ -314,6 +314,155 @@
+  fill: none;
+}
+
+.shell-docs-intelligence-entry:hover,
+.shell-docs-intelligence-entry:focus-visible {
+  background: var(--accent);
+  color: #fff;
+}
+
+.shell-docs-intelligence-entry:hover svg,
+.shell-docs-intelligence-entry:focus-visible svg {
+  fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)